openvdb/points/AttributeSet.cc

```diff
@@ -1353,10 +1369,119 @@ AttributeSet::Descriptor::read(std::istream& is)
 }
 
 
+////////////////////////////////////////
+
+// AttributeSet::Info implementation
+
+
+AttributeSet::Info::Info(const AttributeSet::Descriptor::Ptr& descriptor)
+    : mDescriptor(descriptor)
+    , mArrayInfo(descriptor->size(), Array())
+{
```

Comment:
What's the point on this constructor? AS far as I can tell it currently serves no purpose and makes the API more confusing when using this class.

Reply:
There's an `AttributeSet(const Descriptor::Ptr& descriptor)` constructor. Introducing a matching constructor here means that all AttributeSet construction can be done from an Info instance which keeps the logic in one place.

I consider it an advanced option to build and manipulate an Info class yourself for AttributeSet construction. For most use cases, I expect users to continue to use the other AttributeSet constructors directly.